otx/cli/tools/build.py

```diff
@@ -91,13 +91,13 @@ def main():
     config_manager = ConfigManager(args, mode="build")
     if args.task:
         config_manager.task_type = args.task.upper()
-    if args.work_dir:
-        config_manager.workspace_root = Path(args.work_dir)
+    if args.workspace:
```

Comment:
Is this code necessary? As far as I understand, init function of `ConfigManager` class set it by itself.

Reply:
In this code, workspace_root is not passed in init.

Reply:
I fixed this. https://github.com/openvinotoolkit/training_extensions/pull/1946/commits/2cb8133579f0f6a36f78f4a32ef2a5287d1008e8